invenio_rdm_records/oaiserver/services/services.py

```diff
@@ -33,9 +34,10 @@
 class OAIPMHServerService(Service):
     """OAI-PMH service."""
 
-    def __init__(self, config):
+    def __init__(self, config, reserved_prefix="community-"):
```

Comment:
I don't think this should be configurable via a constructor params, given that it should contain a list of reserved prefixes. We might want to allow extend it only.
What about:
1. converting this to a `set()`, in case there will be multiple prefix to reserve
2. hardcode this list in the service or in the service config (in case we really want to have it configurable...):
```
class OAIPMHServerService(Service):
    RESERVED_PREFIXES = {"community-"}
    def __init__(self, config, extra_reserved_prefixes={}):
        ....
        self.reserved_prefixes = RESERVED_PREFIXES.union(extra_reserved_prefixes)
```
or
```
class OAIPMHServerServiceConfig(ServiceConfig):
    reserved_prefixes = {"community-"}
```

Reply:
Right, thanks a lot!